collector/diskstats_linux.go

```diff
@@ -29,6 +29,9 @@ import (
 
 const (
 	secondsPerTick = 1.0 / 1000.0
+
+	// Read sectors and write sectors are the "standard UNIX 512-byte sectors, not any device- or filesystem-specific block size.
```

Comment:
Can you add a link to `https://www.kernel.org/doc/Documentation/block/stat.txt` here?